build/build.sh

```diff
@@ -39,6 +39,7 @@ pandoc --verbose \
   --include-in-header=build/assets/analytics.html \
   --include-after-body=build/assets/anchors.html \
   --include-after-body=build/assets/hypothesis.html \
+  --include-after-body=build/assets/manubot.html \
```

Comment:
Is the addition of manubot.html to split the CSS that comes from the original gist and the manubot specific changes (as per #156)?

That is fine by me, but what do we do when there are mistakes in `github-pandoc.css`. It seems like modifying this file is a bit of an inevitability.

I also see this change will result in style information being directly inserted into the HTML output, rather than as a standalone `.css` file. There are some pros and cons to this? One pro is that we switch all CSS to being including in HTML output and get rid of having to manage `github-pandoc.css` as separate files on the gh-pages branch. However, this may increase the size of the git repo, since `github-pandoc.css` blobs only need to be stored once.